src/clj/game/core/shuffling.clj

```diff
@@ -21,6 +23,30 @@
     (swap! state update-in [:stats side :shuffle-count] (fnil + 0) 1)
     (swap! state update-in [side kw] shuffle)))
 
+(defn shuffle-cards-into-deck!
+  "Shuffles a given set of cards into the deck. Will print out what's happened. Will always shuffle."
+  [state from-side card shuffle-side & targets]
+  (let [targets (set (keep #(get-card state %) (flatten targets)))
+        targets (filter #(if (= (:zone %) [:discard]) (not (zone-locked? state shuffle-side :discard)) true) targets)
+        lhs (str " uses " (:title card)
+                 (when-not (= from-side shuffle-side)
+                   (str " to force the " (clojure.string/capitalize (name shuffle-side))))
```

Comment:
```suggestion
                   (str " to force the " (str/capitalize (name shuffle-side))))
```

lol i expect there's a require line for this but if not: `[clojure.string :as str]` plz